collect_app/src/main/java/org/odk/collect/android/adapters/RankingListAdapter.java

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.odk.collect.android.adapters;
+
+import android.content.Context;
+import android.graphics.drawable.GradientDrawable;
+import android.support.annotation.NonNull;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.TextView;
+
+import org.javarosa.core.model.SelectChoice;
+import org.javarosa.form.api.FormEntryPrompt;
+import org.odk.collect.android.R;
+import org.odk.collect.android.application.Collect;
+import org.odk.collect.android.utilities.ThemeUtils;
+
+import java.util.Collections;
+import java.util.List;
+
+public class RankingListAdapter extends RecyclerView.Adapter<RankingListAdapter.ItemViewHolder> {
+
+    private Context context;
```

Comment:
I don't think you need this context field. It's only used in `onCreateViewHolder` but you could use `parent.getContext()` there I believe.

Reply:
Fixed.